ansible/roles/prometheus/defaults/main.yml

```diff
@@ -43,5 +43,17 @@ prometheus_configuration:
       static_configs:
         - targets: ['localhost:9100']
 
-prometheus_rules:
-  groups: []
+prometheus_rules: |
+  {% raw %}
+  groups:
+    - name: node
+      rules:
+        - alert: node/systemd-unit-failed
+          # expr: node_systemd_unit_state{state="failed", name!="openipmi.service", name!="nvmf-autoconnect.service"} != 0
```

Comment:
Do you want to keep that commented out expression ?a

Reply:
Yes, it's intentional so we can just comment it back in and remove the other one once we've verified Alertmanager works.